watertap/costing/units/ion_exchange.py

```diff
@@ -221,7 +230,9 @@ def cost_ion_exchange(blk):
     """
     make_capital_cost_var(blk)
     make_fixed_operating_cost_var(blk)
+    ion_exchange_params = blk.costing_package.ion_exchange
     # Conversions to use units from cost equations in reference
+    tot_num_col = blk.unit_model.number_columns + blk.unit_model.number_columns_redund
```

Comment:
It would be nice to have some of these as expressions for the user to access directly in the model results. For example, total number of columns seems like something that could be of interest to a user and would fit well under the `_get_performance_contents` method in the IX unit model.